scene/main/node.cpp

```diff
@@ -2844,6 +2832,48 @@ void Node::remap_nested_resources(Ref<Resource> p_resource, const HashMap<Ref<Re
 }
 #endif
 
+// Duplicates properties that store a Node.
+// This has to be called after nodes have been duplicated since
+// only then do we get a full picture of how the duplicated node tree looks like.
+void Node::_duplicate_properties_node(const Node *p_root, const Node *p_original, Node *p_copy) const {
+	List<PropertyInfo> props;
+	p_copy->get_property_list(&props);
+	StringName script_property_name = CoreStringNames::get_singleton()->_script;
+	for (const PropertyInfo &E : props) {
+		if (!(E.usage & PROPERTY_USAGE_STORAGE)) {
+			continue;
+		}
+		String name = E.name;
+		if (name == script_property_name) {
```

Comment:
I think this check is irrelevant, because you are casting to Node, so a Script won't pass anyway.

Reply:
Removed the if statement

Reply:
Now `script_property_name` is unused. You should remove it too.

Reply:
Done, how silly of me